btf/types.go

```diff
@@ -400,9 +404,11 @@ func (r *Restrict) copy() Type {
 
 // Func is a function definition.
 type Func struct {
-	Name    string
-	Type    Type
-	Linkage FuncLinkage
+	Name      string
+	Type      Type
+	Linkage   FuncLinkage
+	Tags      []string
+	ParamTags [][]string
```

Comment:
This needs a bit of inline documentation around the expectations of the length of the outer/inner arrays. And when will ParamTags be != nil?